integration/shareddata/composites.go

```diff
@@ -66,13 +66,27 @@ var DocumentsDoubles = &Values[string]{
 	handlers: []string{"pg", "tigris"},
 	validators: map[string]map[string]any{
 		"tigris": {
-			"$tigrisSchemaString": tigrisSchema(`"type": "object", "properties": {"v": {"type": "number"}}`),
+			"$tigrisSchemaString": `{
+				"title": "%%collection%%",
+				"primary_key": ["_id"],
+				"properties": {
+					"v": {
+						"type": "object", 
+						"properties": {
+							"foo": {"type": "object", "properties": {"bar": {"type": "string"}} }, 
```

Comment:
Silly question, do we `{"bar": {"type": "string"}}` so we can set `nil` to the value of `foo`?

Reply:
Great question! The answer is "yes", I wanted to add a test case where we can set an object with properties as nil.

Reply:
fixed

Reply:
educational, thanks!